### ISSUE TITLE
⌨ Arrow navigation not working in account sub-menu

### ISSUE THREAD

**Issue by vd1992** [GCTC-NTGC/gc-digital-talent#11991]
## ⌨ Accessibility Issue

Cannot arrow key to "Sign out"
That item has to be tabbed to and is highlighted funny

## 📋 Steps to Reproduce

1. Logged in as manager in this case, effects other users too
2. Tab with keyboard to reach sub-menu
3. Try to reach "Sign out"

## 📸 Screenshot

![image](https://github.com/user-attachments/assets/aaeda2b1-6f23-4ac3-a599-8364246c2486)

![image](https://github.com/user-attachments/assets/78662c3f-3cec-440d-8724-62588144f737)

## ✅ Acceptance Criteria

- [ ] Can keyboard navigate normally through sub-menu
